plugins/linden.py

```diff
@@ -195,7 +207,13 @@ def invest_list_cmd(user_id, args):
     for quote in quotes:
         holding = user['investments'][quote['Symbol']]
         quant, orig_value = holding['quantity'], holding['orig_value']
-        bid = float(quote['Bid'])
+
+        try:
+            bid = float(quote['Bid'])
```

Comment:
This is the bit where it's needed, I assume if you've purchased before then there is a valid price but for some reason that's not true. You'd be better off checking if it's None and adding some message to the output rather than crashing it entirely tho